doc/sphinx/developer_manual/contributing_to_the_documentation/documentation_style_guide.md

```diff
@@ -2,6 +2,164 @@
+3. Adding at least a link to the world bulider file and a grid file to visualize it, preferably also a link to a `.pvtu` file which allows to get the same output as the figure, if present.
+4. Add a figure of the output.
+
+Below is an example code block.
+
+:::::::{code-block} md
+
+::::::{tab-set}
+
+:::::{tab-item} Important lines
+:sync: Partial
+
+```{literalinclude} ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:language: json
+:lineno-start: 5
+:lines: 5-15
+:emphasize-lines: 3,8
+```
+::::{grid} 3
+:::{grid-item-card} BST_05_limit_depth.wb
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:::
+:::{grid-item-card} BST_05_limit_depth.grid
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.grid
+:::
+:::{grid-item-card} Paraview v1 state file 
+:link: ../../_static/paraview_state_files/BST_v1.pvsm
+:::
+::::
+:::::
+
+:::::{tab-item} Full file
+:sync: Full
+
+
+```{literalinclude} ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:language: json
+:lineno-start: 1
+:emphasize-lines: 7,12
+```
+
+::::{grid} 3
+:::{grid-item-card} BST_05_limit_depth.wb
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:::
+:::{grid-item-card} BST_05_limit_depth.grid
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.grid
+:::
+:::{grid-item-card} Paraview v1 state file 
+:link: ../../_static/paraview_state_files/BST_v1.pvsm
+:::
+::::
+:::::
+
+::::::
+
+
+```{figure} ../../../../doc/sphinx/_static/images/user_manual/basic_starter_tutorial/BST_05.png
+:name: BST_05_example
+:alt: Basic Starter Tutorial section 5. 
+:align: center
+
+Image description
+```
+:::::::
+
+
+which looks like this when rendered:
+
+
+::::::{tab-set}
+
+:::::{tab-item} Important lines
+:sync: Partial
+
+```{literalinclude} ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:language: json
+:lineno-start: 5
+:lines: 5-15
+:emphasize-lines: 3,8
+```
+::::{grid} 3
+:::{grid-item-card} BST_05_limit_depth.wb
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:::
+:::{grid-item-card} BST_05_limit_depth.grid
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.grid
+:::
+:::{grid-item-card} Paraview v1 state file 
+:link: ../../_static/paraview_state_files/BST_v1.pvsm
+:::
+::::
+:::::
+
+:::::{tab-item} Full file
+:sync: Full
+
+
+```{literalinclude} ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:language: json
+:lineno-start: 1
+:emphasize-lines: 7,12
+```
+
+::::{grid} 3
+:::{grid-item-card} BST_05_limit_depth.wb
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.wb
+:::
+:::{grid-item-card} BST_05_limit_depth.grid
+:link: ../../_static/gwb_input_files/BST_05_limit_depth.grid
+:::
+:::{grid-item-card} Paraview v1 state file 
+:link: ../../_static/paraview_state_files/BST_v1.pvsm
+:::
+::::
+:::::
+
+::::::
+
+
+```{figure} ../../../../doc/sphinx/_static/images/user_manual/basic_starter_tutorial/BST_05.png
+:name: BST_05_example
+:alt: Basic Starter Tutorial section 5. 
+:align: center
+
+Image description
```

Comment:
Add a newline at the end.

Reply:
between Image description and the ``` or at the end of the document?

Reply:
My diff view shows that there is no newline at the end of the file, which I think is neater to have.